oga/parsing.py

```diff
@@ -86,6 +93,10 @@ def parse_asset(asset_id: str, data: bytes) -> dict:
         "favorites": favorites,
         "files": files}
 
+    if attribution != None:
+        ret["attribution"] = attribution
+
+    return ret
```

Comment:
I have a slight preference for the parser including every field, even if it's `None`.  It's a signal that the parser looked but didn't find it, whereas a new field would be omitted entirely by an older parser.

```suggestion
        "files": files,
        "attribution": attribution}

    return ret
```